specs.md

```
@@ -0,0 +1,94 @@
+**Verification:**
+```bash
+python3 -c "import yaml; yaml.safe_load(open('.github/workflows/ci.yml'))"
+python3 -c "import yaml; yaml.safe_load(open('.github/workflows/publish.yml'))"
+python3 -c "import yaml; yaml.safe_load(open('.github/workflows/sarif-example.yml'))"
+python3 -c "import yaml; yaml.safe_load(open('.github/workflows/diffguard.yml'))"
+```
+
+### AC3: `dtolnay/rust-toolchain` uses specific version
+The `dtolnay/rust-toolchain` action must reference a specific Rust version (e.g., `@1.85.0`), not the `@stable` branch reference.
+
+### AC4: Changes committed to correct branch
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)